### DISCUSSION TITLE
Welcome to wxhelper Discussions!

### DISCUSSION THREAD

**Discussion by ttttupup** [ttttupup/wxhelper#64] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iamtryit** [comment]
为啥提示注入成功了但把微信搞崩了，试了几个版本都这样

**Comment by chengzaizai00** [comment]
能增加  群成员增加 减少的hook消息吗
